test/edu-benefits/1990e/config/sponsorVeteran.unit.spec.jsx

```diff
@@ -60,8 +60,7 @@ describe('Edu 1990e sponsorVeteran', () => {
         value: 'Veteran'
       }
     });
-
-    ReactTestUtils.Simulate.change(find('#root_veteranSocialSecurityNumber'), {
+    ReactTestUtils.Simulate.change(find('#root_view:veteranId_veteranSocialSecurityNumber'), {
```

Comment:
I was just now spelunking to find how other view: type elements were tested.  THanks! Will try this.